package.json

```diff
@@ -35,17 +37,20 @@
   "devDependencies": {
     "@babel/core": "^7.7.4",
     "@babel/preset-env": "^7.9.6",
+    "@babel/preset-typescript": "^7.18.6",
     "@types/jest": "^26.0.24",
     "babel-loader": "^8.1.0",
     "babel-preset-env": "^1.7.0",
     "cypress": "^7.7.0",
     "eslint-plugin-jest": "^24.3.6",
     "jest": "^26.6.3",
+    "typescript": "^4.7.4",
     "webpack": "^4.41.2",
     "webpack-cli": "^3.0.8",
     "webpack-dev-server": "^3.11.0"
   },
   "dependencies": {
+    "@types/quill": "^2.0.9",
```

Comment:
I think this should be a `devDependency`, as it is only needed during development.

Reply:
Oh, I was wondering about that.

Maybe Typescript does this all for us, but my thinking was that since we expose our types, and some of those types come from `@types/quill`, it should be included as a dependency.

I'll try on my computer once I'm at home

Reply:
I'm pretty sure we should keep it in the dependencies, since in a vacuum, the types for Quill won't be there

On devDependencies
![image](https://user-images.githubusercontent.com/2341575/185707726-eb7b1c74-4fb1-4182-a74c-b3af01301ba0.png)

on dependencies
![image](https://user-images.githubusercontent.com/2341575/185707955-347086ac-874e-4f5a-9dbf-00fe237dcabe.png)

Even though I don't know how they can use the plugin without Quill, I don't think it's up to us to assume they would already have the `@types/quill`